### PR TITLE
Add recommended alert bicep templates

### PR DESCRIPTION
**Pull request by carlin-q-scott**
Add bicep templates with missing property added to address #920

### CODE REVIEW

**Comment by carlin-q-scott** [comment]
@microsoft-github-policy-service agree [company="ancora Software, Inc."]

**Comment by carlin-q-scott** [comment]
@microsoft-github-policy-service agree company="ancora Software, Inc."

File: alerts/recommended_alerts_ARM/ContainerCPUPercentage.bicep [hunk 1]
```diff
@@ -0,0 +1,123 @@
+@description('Name of the alert')
```
Comment (ganga1980) [inline alerts/recommended_alerts_ARM/ContainerCPUPercentage.bicep:1]: Thanks, @carlin-q-scott  for making the PR to add the bicep template support for the recommended alerts.  Did you test these bicep templates works as expected?

Comment (carlin-q-scott) [reply alerts/recommended_alerts_ARM/ContainerCPUPercentage.bicep:1]: I'm using these ones in my deployment without issue:

- NodeCPUPercentage.bicep
- NodeDiskUsagePercentage.bicep
- NodeWorkingSetMemoryPercentage.bicep
- PodsInFailedState.bicep
- RestartingContainerCount.bicep
- StaleJobsCount.bicep

I did not verify the other ones, but these are barely modified from what was generated using `az bicep decompile`.

**Review by ganga1980 — APPROVED** [review]

**Comment by ganga1980** [comment]
Hi, @carlin-q-scott , can you please make PR against ci_prod branch which is our default branch?  We dont use ci_dev branch.

**Comment by carlin-q-scott** [comment]
@ganga1980 I used the dev branch because it was ahead of prod by quite a few commits. But I see prod has been committed to more recently, so I can re-target this PR.

I haven't enabled Prometheus in my cluster because it's in preview and isn't at feature parity with Container Insights from what I can tell. Creating bicep templates for those alerts seems like it would be out of scope for this PR but it's really easy for me to do that conversion if you're asking for another PR for the Prometheus alert ARM templates.

**Pull request closed without merging**